Zilon.Core/Zilon.BotMassLauncher/Program.cs

```diff
@@ -63,6 +75,20 @@ static void Main(string[] args)
             Console.WriteLine($"[x] COMPLETE");
         }
 
+        private static string GenerateUniquePreffix()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'GenerateUniquePreffix'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5310892)